nf_core/lint.py

```diff
@@ -197,7 +201,7 @@ def check_config_vars(self):
             'timeline.file',
             'trace.file',
             'report.file',
-            'process.container',
+            'params.container',
```

Comment:
Maybe both? `process.container` is the important one, `params.container` is just to move the declaration next to the version so that it's easier to update..

Reply:
shall we let linting fail, when `process.container` is not defined? Cause we made container usage pretty a core thing of nf-core

Reply:
and warning for missing `params.container`?

Reply:
Or maybe it is easier to make both mandatory. And I can check if `process.container == params.container` and then continue with the versioning check

Reply:
Not a failure, no. Pipelines may not be able to use a single container for the entire workflow, in which case `container` will need to be specified at process level instead of global process scope.